rules/msk_topic_config.go

```diff
@@ -202,7 +208,7 @@ func (r *MSKTopicConfigRule) validateCompressionType(
 			},
 		)
 		if err != nil {
-			return fmt.Errorf("emitting issue with fix: no replication factor: %w", err)
+			return fmt.Errorf("emitting issue with fix: no compression type: %w", err)
```

Comment:
Why at some places you return just err, and in some other- you wrap it?

Reply:
This is driven mostly by the wrapcheck.

Also, if the error is already wrapped, we shouldn't need to wrap it again, as the information should be enough to pinpoint the throwing line.
For when using gohcl.DecodeExpression I think the return diags should be enough to figure out where the issue is coming from.

Reply:
How will we know, which topic config is wrong? We sometimes are passing this information in [other checks](https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/blob/18ccb31e27b9624add399f084eed10458169df1d/rules/msk_topic_name.go#L106)

Reply:
I see your point and it is valid.

On the other hand we don't expect these to error, and we haven't seen it error so far. 
I can come back to all these errors and add more context on them, but I'd rather see how it goes mid-term, and if we actually need more info on it.